src/main/java/codesquad/banchan/domain/Banchan.java

```diff
@@ -0,0 +1,42 @@
+package codesquad.banchan.domain;
+
+import codesquad.bestcategory.domain.BestCategory;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Banchan {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subTitle;
```

Comment:
hey guys! subTitle is not two word. you need to merge this words. Horrible~

Reply:
o..okay; 😭